### PR TITLE
Apachecon workshop

### PR DESCRIPTION
**Pull request by ei8fdb**

### CODE REVIEW

**Comment by belenbarrospena** [comment]
It looks like this PR should include only commit [490322018cff845a330b79616b2ffd1f90ca2444](https://github.com/opensourcedesign/events/pull/131/commits/490322018cff845a330b79616b2ffd1f90ca2444), since the FOSDEM 2019 pages already seem to exist (hence the conflicts). Am I missing something?

**Comment by ei8fdb** [comment]
@belenbarrospena can you look now? I've deleted the 2 files, except for the apachecon workshop file.

**Review by evalica — DISMISSED** [review]

**Review by ZoranPandovski — DISMISSED** [review]

File: _posts/2019-02-02-opensourcedesign-fosdem.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: event
+title:  "Call for Participation: Open Source Design Devroom at FOSDEM 2019"
```
Comment (jancborchardt) [inline _posts/2019-02-02-opensourcedesign-fosdem.md]: This is duplicated and already exists, no?

File: _posts/2019-10-22-apachecon-eu-opensource-design.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+layout: event
+title:  "Call for Participation: Open Source Design track at ApacheCon Europe 2019"
```
Comment (jancborchardt) [inline _posts/2019-10-22-apachecon-eu-opensource-design.md]: We can cut the "Call for Participation" part here as that’s over, right?

**Comment by jancborchardt** [comment]
@ei8fdb could you check and do the adjustments? :)

File: _posts/2019-10-22-apachecon-eu-opensource-design.md (hunk 2 above)
Comment (jancborchardt) [reply _posts/2019-10-22-apachecon-eu-opensource-design.md]:
```suggestion
title:  "Open Source Design track at ApacheCon Europe 2019"
```
@ei8fdb you could accept and commit that suggestion, then it simply turns the CfP to the actual event. :)

File: _posts/2019-10-25-apachecon-eu-opensource-design-usability-workshop.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+layout: event
+title:  "Cheap, easy and illuminating – how to plan, run and analyse usability studies workshop"
+date:   2019-10-25
+categories: conference , workshop
+eventDate: Fri, 25 Oct 2019
+location: Room yet to be confirmed at c-base, Rungestraße 20, 10179 Berlin, Germany
+time: 10:00 - 14:00
+status: upcoming
```
Comment (jancborchardt) [inline _posts/2019-10-25-apachecon-eu-opensource-design-usability-workshop.md]:
```suggestion
status: past
```

File: _posts/2019-10-22-apachecon-eu-opensource-design.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+date:   2019-10-22
+categories: conference
+eventDate: 22  - 24 Sat, 2 Feb 2019
+location: Room not yet confirmed, Kulturbrauerei, Schönhauser Allee 36 10435 Berlin, Germany
+time: full days
+status: upcoming
```
Comment (jancborchardt) [inline _posts/2019-10-22-apachecon-eu-opensource-design.md]:
```suggestion
status: past
```

**Review by ei8fdb — COMMENTED** [review]
Deleted the dupe for fosdem-2019. Apologies - I got busy and didn't get a chance to fix it.

**Review by ZoranPandovski — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**